webapp/channels/src/components/advanced_text_editor/advanced_text_editor.tsx

```diff
@@ -457,14 +457,19 @@ const AdvanceTextEditor = ({
                         role='application'
                         id='advancedTextEditorCell'
                         data-a11y-sort-order='2'
-                        aria-label={Utils.localizeMessage(
-                            'channelView.login.successfull',
-                            'Login Successfull',
-                        ) + ' ' + ariaLabelMessageInput}
+                        aria-describedby='advancedTextEditorCellAriaLabel'
                         tabIndex={-1}
                         className='AdvancedTextEditor__cell a11y__region'
                     >
                         {labels}
+                        <label
+                            id='advancedTextEditorCellAriaLabel'
+                            style={{display: 'none', visibility: 'hidden', height: 0}}
```

Comment:
Instead of passing a style object, could you change this to use the `sr-only` CSS class? That's typically what we do for elements like this